source/reference/tables.txt

```diff
@@ -0,0 +1,132 @@
+.. _tables-reference:
+
+======
+Tables
+======
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/styles/corrections.rst
+
+Tables compare items across multiple domains.
```

Comment:
```suggestion
Tables help compare items across multiple domains or present information in a ___ manor.
```
Suggestion: I think tables are also used a lot when describing parameters for an API or function -- so I think its worth incorporating that here.

Reply:
Added in my own commit!